core/src/main/java/org/elasticsearch/common/geo/builders/ShapeBuilder.java

```diff
@@ -77,16 +77,10 @@
     /** @see com.spatial4j.core.shape.jts.JtsGeometry#index() */
     protected final boolean autoIndexJtsGeometry = true;//may want to turn off once SpatialStrategy impls do it.
 
-    protected Orientation orientation = Orientation.RIGHT;
-
     protected ShapeBuilder() {
```

Comment:
Looked into this again, added checks that ensure that JTS and Spatial4J are present when registering the Shape builders, the GeoShapeQueryBuilder and Mapper. It looks like we only checked for JTS beeing present so far, but if JTS is there but Spatial4J missing, registering the ShapeBuilders/GeoShapeQuery also fails because of the static reference to JtsSpatialContext somehow. I added another commit that should prevent this.